docs/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
I think it's recommended to specify a particular version to avoid breakage. Then the Manifest isn't needed.

Reply:
👍

Reply:
Manifest is useful becaue then you can include in it
```
[[Tables]]
path = ".."
uuid = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
```

which makes sure the docstrings are right for the dev version docs.